app/controllers/searches_controller.rb

```diff
@@ -204,7 +204,7 @@ def favoriting
   def inactive_friends
     @user_items = build_user_items(@searched_tw_user.inactive_friends)
     @graph = @searched_tw_user.inactive_friends_graph
-    @tweet_text = view_context.inactive_friends_text(@user_items.slice(0, 3).map{|i| i[:target] }, @searched_tw_user)
+    @tweet_text = view_context.inactive_friends_text(@user_items.slice(0, 3).map { |i| i[:target] }, @searched_tw_user)
```

Comment:
__[rubocop]__
* Line is too long. [89/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)